### PR TITLE
Th/installation instructions updates

### PR DESCRIPTION
**Pull request by tianhe**
screen shot 2017-09-25 at 8 06 46 pm
screen shot 2017-09-25 at 8 06 34 pm
screen shot 2017-09-25 at 8 06 23 pm

### CODE REVIEW

**Comment by tianhe** [comment]
Here are some updates to the organization of instructions and additional content that I think maybe helpful for a person who is setting up Livepeer for the first-time.  Inspired by my experience last week!

File: README.md [hunk 1]
```diff
@@ -14,57 +14,73 @@ Building and running this node allows you to:
...
-The easiest way to install Livepeer is by downloading it from the [release page on Github](https://github.com/livepeer/go-livepeer/releases).  Pick the appropriate platform and the latest version.
+## Installing Livepeer
+### Option 1: Download executables
+The easiest way to install Livepeer is by downloading the `livepeer` and `livepeer_cli` executables from the [release page on Github](https://github.com/livepeer/go-livepeer/releases). 
 
-## Build
-If you have never set up your Go programming environment, do so according to Go's [Getting Started Guide](https://golang.org/doc/install).
+1. Download the packages for your OS - darwin for Macs and linux for linux. 
+2. Rename them to `livepeer` and `livepeer_cli`
+3. Make sure they have executable permissions by running `chmod +x livepeer` and `chmod +x livepeer_cli`
```
Comment (dob) [inline README.md:23]: Note to self that we should redo the process for generating these binaries to just have one linux and one windows .tar.gz that contains the already permissioned livepeer and livepeer_cli files. Then we can get rid of the renaming and the chmod'ing.

File: README.md [hunk 2]
```diff
@@ -14,57 +14,73 @@ Building and running this node allows you to:
...
 ### Quick start
-- Make sure you have done all the setup steps. (Have `ffmpeg` installed, have `geth` running on the Livepeer private testnet)
+- Make sure you have successfully gone through the steps in 'Installing Livepeer' and 'Additional Dependencies'.
 
-- Run `livepeer -testnet` to start Livepeer. Make sure you have the `livepeer` executable.  It can be downloaded from the [releases page](https://github.com/livepeer/go-livepeer/releases) (you should download the packages for your OS - darwin for Macs and linux for linux.  You should also rename them to `livepeer` and `livepeer_cli`, and make sure they have executable permissions).
+- Start `geth ` (see step 4 of 'Livepeer private Ethereum testnet').
 
-- Run `livepeer_cli`. You should see a wizard launch in the command line. Make sure you installed the `livepeer_cli` executable.  It can be downloaded from the [releases page](https://github.com/livepeer/go-livepeer/releases).
+- Run `./livepeer -testnet`.
+  * Take note of the log output for your Eth address.  You should see a line: `'Using Eth account: ...'`
```
Comment (dob) [inline README.md]: The ETH address is also printed out when you run `livepeer_cli`, so perhaps that's easier to find than requesting that they watch log output?

**Review by dob — COMMENTED** [review]
Thank you for these updates. Much simpler and the formatting is more clear.

File: README.md (hunk 1 above)
Comment (dob) [reply README.md:23]: tracked here: https://github.com/livepeer/go-livepeer/issues/131

File: README.md (hunk 2 above)
Comment (tianhe) [reply README.md]: @dob good point! pushed an update with instructions directing users to use livepeer_cli to access info

**Comment by ericxtang** [comment]
This is awesome.  Thanks @tianhe!

**Pull request merged**